dask_cuda/cli/dask_config.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+
+import click
+
+from distributed import Client
+from distributed.preloading import validate_preload_argv
+from distributed.security import Security
+
+from ..utils import print_cluster_config
+
+logger = logging.getLogger(__name__)
+
+
+pem_file_option_type = click.Path(exists=True, resolve_path=True)
+
+
+@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.argument("scheduler", type=str, required=False)
+@click.argument(
+    "preload_argv", nargs=-1, type=click.UNPROCESSED, callback=validate_preload_argv
+)
```

Comment:
These arguments don't seem to go anywhere? [...], oh I see, `validate_preload_argv` side-effectfully imports the specified module. Do we need this here?